src/Components/Aspire.OpenAI/Aspire.OpenAI.csproj

```
@@ -0,0 +1,30 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetCurrent)</TargetFramework>
```

Use this property now which is set to `net8.0`, because `NetCurrent` is now `net9.0`.

```suggestion
    <TargetFramework>$(DefaultTargetFramework)</TargetFramework>
```